inductiva/fluids/scenarios/fluid_block/fluid_block.py

```diff
@@ -116,22 +115,12 @@ def simulate(
         return SPHSimulationOutput(output_path)
 
     @singledispatchmethod
-    @classmethod
-    def get_config_filename(cls, simulator: Simulator):  # pylint: disable=unused-argument
-        raise ValueError(
-            f"Simulator not supported for `{cls.__name__}` scenario.")
+    def get_config_filename(self, simulator: Simulator):
```

Comment:
This is only required when we have multiple simulators right? We could have a dict for this when we define the valid_simulators.

Reply:
Nop, this is required when there is a main config file for the simulation, which is the case for SPlisHSPlasH, DualSPHysics, or SWASH. This is used to pass a parameter to the backend that specifies which file in the input dir is this config file.